Decisions.Microsoft365.Exchange/Steps/EmailSteps.cs

```diff
@@ -164,8 +164,8 @@ public string MarkEmailAsRead(ExchangeSettings? settingsOverride, string userIde
 
             return response.StatusCode.ToString();
         }
-        
-        private Microsoft365Recipient[]? GetRecipients(string[] emailAddresses)
+
+        protected virtual Microsoft365Recipient[]? GetRecipients(string[] emailAddresses)
```

Comment:
Why do we need to make this `virtual`?

Reply:
We don't, I meant to change that back. My bad